packages/cli/lib/cmds/optimize.js

```diff
@@ -26,10 +26,12 @@ class OptimizeCmd {
   }
   async run(args, logger) {
     const urlOrPath = args._[1];
-    const {host, rtv, lts, geoapi} = args;
+    const {host, rtv, lts, geoapi, sxgHost, baseDomain} = args;
     const html = await this.load_(urlOrPath);
     const optimized = await this.optimizer_.transformHtml(html, {
       ampUrlPrefix: host,
+      baseDomain: baseDomain,
+      rewriteUrlsToAmpCache: sxgHost,
```

Comment:
As in the readme, basedomain is the original domain you'd host your content on (for when you pass a relative path to `amp-img`). 

`rewriteUrlsToAmpCache` is going to be the `sxgHost`, i.e. `cdn.ampproject.org`

Reply:
The naming is a bit ambiguous - it reads as an action, rather than a label. would `ampCacheURL` fit a bit better, or am I misunderstanding its purpose?